pkg/lib/aws/gen_resource_metadata.py

```diff
@@ -307,6 +351,20 @@ def main():
                 }
             )
 
+        ebs_metadatas_str = ""
+
+        for ebs_type in sorted(ebs_metadata.keys()):
+            metadata = ebs_metadata[ebs_type]
+            ebs_metadatas_str += ebs_type_map_template.substitute(
+                {
+                    "region": region,
+                    "type": ebs_type,
+                    "priceGB": metadata["priceGB"],
+                    "priceIOPS": metadata["priceIOPS"],
```

Comment:
nit: this should probably use underscores to maintain Python conventions (you will have to change the template above too):

```python
"price_gb": metadata["price_gb"],
"price_iops": metadata["price_iops"],
```